hack/sandboxed.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:alpine as builder
+
+WORKDIR /go/src/sigs.k8s.io/krew
+
+# build binary
+COPY . .
+RUN go build -ldflags "-s -w" ./cmd/krew
+
+# production image
+FROM alpine
+RUN apk add git
```

Comment:
you can do `apk add --no-cache git` for smaller image
also I think you'll need `ca-certificates` as well as `kubectl` in this image.

Reply:
`ca-certificate` is a dependency of `git` and already installed (just checked).

Adding `kubectl` is a good idea.